### ISSUE TITLE
[5.1.0-beta] `react-native link` gets stuck

### ISSUE THREAD

**Issue by fungilation** [Microsoft/react-native-code-push#999]
### Steps to Reproduce
1. `react-native unlink react-native-code-push`
2. `react-native link react-native-code-push`
3. Doesn't return to cli, stuck on last line: "codepush.gradle" is already linked in the build definition

### Expected Behavior
Exit cleanly on `link`

### Actual Behavior
See 3 above

### Reproducible Demo
Output copied from terminal, with [STRINGS] replaced:
```
$ react-native unlink react-native-code-push
Scanning 611 folders for symlinks in [PROJECT]/node_modules (7ms)
rnpm-install info Unlinking react-native-code-push android dependency 
rnpm-install info Android module react-native-code-push has been successfully unlinked 
rnpm-install info Unlinking react-native-code-push ios dependency 
rnpm-install info iOS module react-native-code-push has been successfully unlinked 
$ react-native link react-native-code-push
Scanning 611 folders for symlinks in [PROJECT]/node_modules (8ms)
? What is your CodePush deployment key for Android (hit <ENTER> to ignore) [KEY]
rnpm-install info Linking react-native-code-push android dependency 
rnpm-install info Android module react-native-code-push has been successfully linked 
rnpm-install info Linking react-native-code-push ios dependency 
rnpm-install info iOS module react-native-code-push has been successfully linked 
"CodePush.h" header already imported.
"jsCodeLocation" already pointing to "[CodePush bundleURL]".
Could not determine correct xcode proj path to retrieve related plist file, there are multiple xcodeproj under the solution.
? What is your CodePush deployment key for iOS (hit <ENTER> to ignore) "getJSBundleFile" is already overridden
"codepush.gradle" is already linked in the build definition
```
... then stuck here, not returning to cli. Have to force quit with CTRL-C

### Environment

* react-native-code-push version: 5.1.0-beta
* react-native version: 0.48.3
* iOS/Android/Windows version: iOS 10.3.3
* Does this reproduce on a debug build or release build? Neither (cli)
* Does this reproduce on a simulator, or only on a physical device? Neither

**Comment by ruslan-bikkinin** [comment]
@fungilation thanks for reaching us! Yeah, it seems to be a bug in postlinking scripts, i will be working on fix tomorrow. If you need additional assistance from our side with linking your application with react-native-code-push feel free to ask.

**Comment by fungilation** [comment]
Thanks for the quick reply! This is an existing installation and CTRL-C after relinking doesn't seem to have broken anything with my build. The broken postlinking scripts may break new installations however.

**Comment by ruslan-bikkinin** [comment]
@fungilation I just made PR with fix, could you please check it out and let us know does it resolve your issue?

**Comment by fungilation** [comment]
@ruslan-bikkinin How should I test it, `yarn add https://github.com/Microsoft/react-native-code-push.git` instead of using npm since there's no new release yet with your PR?

**Comment by ruslan-bikkinin** [comment]
@fungilation you can check out patched version of react-native-code-push directly from my branch:
```shell
cd /path/to/your/rn-app
rm -rf node_modules/react-native-code-push
npm i ruslan-bikkinin/react-native-code-push#fix-postlink
```

**Comment by fungilation** [comment]
Looks like you fixed it:

```
$ yarn remove react-native-code-push
...
$ yarn add ruslan-bikkinin/react-native-code-push#fix-postlink
...
$ react-native link react-native-code-push
Scanning 610 folders for symlinks in [PROJECT]/node_modules (7ms)
? What is your CodePush deployment key for Android (hit <ENTER> to ignore) 
rnpm-install info Linking react-native-code-push android dependency 
rnpm-install info Android module react-native-code-push has been successfully linked 
rnpm-install info Linking react-native-code-push ios dependency 
rnpm-install info iOS module react-native-code-push has been successfully linked 
Running ios postlink script
"CodePush.h" header already imported.
"jsCodeLocation" already pointing to "[CodePush bundleURL]".
Could not determine correct xcode proj path to retrieve related plist file, there are multiple xcodeproj under the solution.
? What is your CodePush deployment key for iOS (hit <ENTER> to ignore) [DSN]
Running android postlink script
"getJSBundleFile" is already overridden
"codepush.gradle" is already linked in the build definition
$
```

Exited to cli prompt cleanly

**Comment by ruslan-bikkinin** [comment]
@fungilation thank you for reply! So fix in the master now and I am closing issue.

**Comment by ethanyuwang** [comment]
I'm still seeing `Could not determine correct xcode proj path to retrieve related plist file, there are multiple xcodeproj under the solution.` After a reinstall. How should I fix it?

**Comment by luongs3** [comment]
still stuck in this issue ?

**Comment by garrottkai** [comment]
Same issue still occurs for me on v5.3.2

**Comment by iTOYS** [comment]
Hi guys,
could you please paste here your terminal output?

Thanks!

**Comment by garrottkai** [comment]
All I'm getting is:
```
? What is your CodePush deployment key for Android (hit <ENTER> to ignore)
Running ios postlink script
? What is your CodePush deployment key for iOS (hit <ENTER> to ignore) 
Running android postlink script
```
It hangs there

**Comment by iTOYS** [comment]
Unfortunately I can't reproduce this issue.
My steps:
```bash
$ react-native init AwesomeProject
$ cd AwesomeProject
$ npm i react-native-code-push --save
$ react-native link react-native-code-push
$ react-native unlink react-native-code-push
$ react-native link react-native-code-push
```
And all works fine.

Could you please give me more clues e.g.: rn version, node version, npm version, additional steps.

**Comment by mouthzipper** [comment]
same issue here.

**Comment by antofa** [comment]
I had this issue too and resolved it by downgrading to node 9.7.0 from 10.9.0.
